csm_web/frontend/src/components/MentorSection.js

```diff
@@ -296,10 +297,12 @@ function zeroPadTwoDigit(num) {
 class SpacetimeEditModal extends React.Component {
   constructor(props) {
     super(props);
+    // Time string comes as HH:MM:ss, TimeInput expects HH:MM
+    let timeString = props.defaultSpacetime.startTime;
```

Comment:
```suggestion
    const timeString = props.defaultSpacetime.startTime;
```
Arguably bikeshedding on my part, but it never changes and the existing code tends to prefer `const`

Reply:
was probably thinking about rust, where let is immutable by default